packages/accounts-base/accounts_client.js

```diff
@@ -307,7 +307,7 @@ export class AccountsClient extends AccountsCommon {
           if (this._tokenExpiresSoon(result.tokenExpires)) {
             this.makeClientLoggedOut();
           } else {
-            this.callLoginMethod({
+            return this.callLoginMethod({
```

Comment:
Why is this return necessary?

Reply:
That's because this invocating is inside a `DDP.onReconnect`, which is a hook. So the idea here was to return this promise and wait for it [here](https://github.com/meteor/meteor/pull/13221/commits/82fc4e7aa4379001c4c3cd4282a4e65da48a02d7#diff-5a0f1c2953dc6856044b4803bb19c0b301386cd8836c878e36f2776d6316dc79R1798-R1799). But I think that won't be necessary anymore.